app/services/renew_book.rb

```diff
@@ -0,0 +1,24 @@
+class RenewBook
+  def self.process(checkout:, user:)
+    new(checkout: checkout, user: user).process
+  end
+
+  def initialize(checkout:, user:)
+    @checkout = checkout
+    @user = user
+  end
+
+  def process
+    book = Book.find(@checkout.book_id)
+    due_date = @checkout.due_on + Checkout::CHECKOUT_PERIOD_IN_DAYS.days
+    book_fine = @checkout.calculate_fine
+    closed_at = DateTime.now
```

Comment:
I wouldn't put this in a var. It's extra stuff to read, but it doesn't make it easier to understand over doing `closed_at: Time.now`